_posts/2020-08-05-cockpit-225.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Cockpit 225 and Cockpit Podman 21
+author: marusak
+date: 2020-08-05
+tags: cockpit linux
+slug: cockpit-225
+category: release
+summary: Cockpit and Cockpit Podman news
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 223 and Cockpit Podman version 21.
+
+### Machines: Add support for VM snapshots
+
+Libvirt supports two types of snapshots, [internal and external](https://libvirt.org/formatsnapshot.html),
+but external are not yet in feature parity with the internal. For this reason, cockpit developers decided to go
+with the internal snapshots in the Cockpit UI. Because of using internal snapshots this feature only works with
+qcow2 disk images. VM snapshots are supported for running VMs, in which case the snapshot will combine disk and
+VM memory state, as well as offline VMs, in which case only the disk contents will be saved.
```

Comment:
```suggestion
Cockpit now supports [libvirt's qcow2-based "internal" snapshotting](https://libvirt.org/formatsnapshot.html).

When a virtual machine is not running and Cockpit makes a snapshot, only the disk contents will be saved. When a virtual machine is running, Cockpit will snapshot both the disk and memory state.
```

Reply:
Removed the comparison of internal versus external, which is too detailed for our audience and didn't make sense. (Yes, they're different _somehow_. But now? Yes, we chose one. Why? We didn't say, and I don't think it's relevant.)

I took the rest of the text and rewrote it.